curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-382-generating-polygons.md

```diff
@@ -10,35 +10,52 @@ dashedName: problem-382-generating-polygons
 
 A polygon is a flat shape consisting of straight line segments that are joined to form a closed chain or circuit. A polygon consists of at least three sides and does not self-intersect.
 
-A set S of positive numbers is said to generate a polygon P if: no two sides of P are the same length, the length of every side of P is in S, and S contains no other value.
+A set $S$ of positive numbers is said to generate a polygon $P$ if:
 
-For example: The set {3, 4, 5} generates a polygon with sides 3, 4, and 5 (a triangle). The set {6, 9, 11, 24} generates a polygon with sides 6, 9, 11, and 24 (a quadrilateral). The sets {1, 2, 3} and {2, 3, 4, 9} do not generate any polygon at all.
+- no two sides of $P$ are the same length,
+- the length of every side of $P$ is in $S$, and
+- $S$ contains no other value.
 
-Consider the sequence s, defined as follows:s1 = 1, s2 = 2, s3 = 3 sn = sn-1 + sn-3 for n > 3.
+For example:
 
-Let Un be the set {s1, s2, ..., sn}. For example, U10 = {1, 2, 3, 4, 6, 9, 13, 19, 28, 41}. Let f(n) be the number of subsets of Un which generate at least one polygon. For example, f(5) = 7, f(10) = 501 and f(25) = 18635853.
+The set {3, 4, 5} generates a polygon with sides 3, 4, and 5 (a triangle).
 
-Find the last 9 digits of f(1018).
+The set {6, 9, 11, 24} generates a polygon with sides 6, 9, 11, and 24 (a quadrilateral).
+
+The sets {1, 2, 3} and {2, 3, 4, 9} do not generate any polygon at all.
```

Comment:
Should these sets be in MathJax?

Reply:
I'd say not necessarily, it's kind of like with coordinates of points ie. (0, 1). It could be written with MathJax, but the difference in look is rather marginal. So it's more a case of preference/consistency, if numbers are not using the sub/superscript and other math notation is not involved. I'm pretty sure this aspect might not be too consistent in the clean-up.

Having said that, if you want I can change it to use MathJax here. What do you think?

Reply:
I'm fine with it either way 🙂. Just thought I'd double check.